src/CommunityVoices/App/Website/View/Identification.php

```diff
@@ -118,10 +123,10 @@ public function postCredentials($request, $result)
 
     public function getLogout($request)
     {
+        $newlink = in_array($request->headers->get('referer'),$this->restricted) ? $this->urlGenerator->generate('root') : $request->headers->get('referer');
```

Comment:
Nice!  I wonder if we could expand to auto redirect whenever we get an AccessDenied from the backend?  Could make this more generic, as the above code defining paths to redirect from, so then we are basically storing access control in multiple places now.  This is maybe out of the scope of this PR, but would help get it to be more good practice.

Reply:
Yes, historically this was not an issue because the API errors would stop the Website side.  Would you like me to try to set up that behavior again?  I think it's a little outside of the scope of your responsibilities, so I can take it on to save you from this frustrating error.

Reply:
Sure! It isn't too high on my priority list though.

Reply:
I'll try to look into this in the next day or two - this would clear up my concerns with the implementation anyhow.  No need to wait on me for merging this, I'll resolve this since I'm writing this down elsewhere.